pkg/apis/nmstate/v1alpha1/nodenetworkconfigurationpolicy_types.go

```diff
@@ -30,36 +48,65 @@ type NodeNetworkConfigurationPolicySpec struct {
 // NodeNetworkConfigurationPolicyStatus defines the observed state of NodeNetworkConfigurationPolicy
 // +k8s:openapi-gen=true
 type NodeNetworkConfigurationPolicyStatus struct {
-	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
-	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
-	// Add custom validation using kubebuilder tags: https://book.kubebuilder.io/beyond_basics/generating_crd.html
+	Nodes NodeInfoList `json:"nodes,omitempty" optional:"true"`
 }
 
-// +genclient
-// +genclient:nonNamespaced
-// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// TODO: This is a temporary solution. This list will be replaced by a dedicated
+// NodeNetworkConfigurationEnactment object.
+type NodeInfoList []NodeInfo
 
-// NodeNetworkConfigurationPolicy is the Schema for the nodenetworkconfigurationpolicies API
-// +k8s:openapi-gen=true
-// +kubebuilder:subresource:status
-// +kubebuilder:resource:path=nodenetworkconfigurationpolicies,shortName=nncp
-type NodeNetworkConfigurationPolicy struct {
-	metav1.TypeMeta   `json:",inline"`
-	metav1.ObjectMeta `json:"metadata,omitempty"`
+type NodeInfo struct {
```

Comment:
What about start to call this NodeNetworkConfigurationEnactment so we start to introduce the Enactment concept ? then we move it to it's own CRD.

Reply:
Done